wls-gui-wahllokalsystem/src/composables/ergebnismeldung/aWerteService.ts

```diff
@@ -0,0 +1,39 @@
+import {
+  AWerteControllerApi,
+  Configuration,
+} from "@/api/wls-clients/generated-ergebnismeldung-api";
+import { useAWerteMapper } from "@/composables/ergebnismeldung/aWerteMapper.ts";
+import { useUserNotificationService } from "@/composables/userNotification/userNotificationService.ts";
+import { ERGEBNISMELDUNG_SERVICE_API_URL } from "@/constants.ts";
+import { UserNotificationCategoryEnum } from "@/types/userNotification/UserNotificationCategoryEnum.ts";
+
+const { toModel } = useAWerteMapper();
+const { addNotification } = useUserNotificationService();
+
+export function useAWerteService() {
+  const aWerteController = new AWerteControllerApi(
+    new Configuration({ basePath: ERGEBNISMELDUNG_SERVICE_API_URL })
+  );
+
+  async function getAWerte(wahlbezirkId: string, sendNotification = true) {
+    try {
+      const response = await aWerteController.getAWerte(wahlbezirkId);
+      if (sendNotification) {
+        addNotification(`AWerte geladen`, UserNotificationCategoryEnum.SUCCESS);
```

Comment:
```suggestion
        addNotification(`AWerte erfolgreich geladen`, UserNotificationCategoryEnum.SUCCESS);
```
Damit es einheitlich zu den anderen Nachrichten ist.